ckanext/qa/tasks.py

```diff
@@ -121,15 +103,13 @@ class FakeRequest:
     registry.register(translator, fakepylons.translator)
 
 
-def update_package(ckan_ini_filepath, package_id):
+def update_package(package_id):
```

Comment:
tasks is called on batch. did you also remove the filepath from it?

how does it load the ckan environment on boot when its in async mode?

Reply:
The batch processes already have a CKAN environment loaded; they're paster/Click instances.

I can't see any reason why we would need to reload from a file. If the entire config object were being passed to the job, then maybe there would be an edge case where it's useful...but if the config has to be on the disk anyway, at the same path as it was on the web server, then it's pointless.

Reply:
ok, this needs to be fully tested in dev then.